boards/arm/mimx8mm_evk_a53/board.cmake

```diff
@@ -0,0 +1 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
need add debugger support
`
board_set_debugger_ifnset(jlink)
board_set_flasher_ifnset(jlink)

board_runner_args(jlink "--device=MIMX8MD6_A53")
include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
`

Reply:
I am not sure, but I not see this "./boards/arm/bcm958402m2_a72/board.cmake" and "boards/arm/qemu_cortex_a53/board.cmake",  Is it really needed?

Reply:
without that you can not use west flash to download the application. how do you program the board now?

Reply:
Use U-Boot to load/kick the zephyr.bin or use Linux to load it for hypervisor usage. zephyr.bin is stored in sdcard fat partition.

Reply:
@MrVan  the official way is to use `west flash`

Reply:
Nope, using U-Boot is fine and `west flash` is totally optional here. Not always you want to use the debugger to load the image also considering that you do not always have a debugger.

Reply:
then user need extra steps to setup uboot, and as uboot is evovled overtime, this needs some extra documents and efforts.

Reply:
How to use U-Boot is already documented in this commit and that is enough. Fine to me if we can use `west flash` to flash the board as well but there is no such a thing as "official" way to flash or use the board. @MrVan is this a problem adding support for JLink?

Reply:
I am not familar with JLink. To me, we have sdcard to hold zephyr.bin, zephyr.bin is copied to the sdcard by mounting the sdcard to Linux PC. Could west flash support writing fat partition of a sdcard? Actually we have yocto Linux for image generating.

Reply:
@MrVan west does not support fat partition write, it is a different way. It may not be officially defined to use west flash, but basically we expect to have an easy to use solution.

Reply:
> @MrVan west does not support fat partition write, it is a different way. It may not be officially defined to use west flash, but basically we expect to have an easy to use solution. Besides if there no debuger tools supported, how users suppose to debug the program? IMHO, it is not a good enough enablement.

I disagree. U-Boot + SD is indeed an easy solution to use in this case (and pretty standard when dealing with AArch64 boards, much more common than having a J-Link available) and you cannot expect that everybody has access to a debugger / J-Link or that every board has a debug port easily accessible. So, this enablement is good enough for merging.

Now, that said @MrVan, can you add what suggested here https://github.com/zephyrproject-rtos/zephyr/pull/32694#discussion_r583531459 so that `west flash` can be used when a debugger is available? This doesn't prevent the user from using U-Boot and the method you already described.

Please, also update the documentation describing both the methods.